### ISSUE TITLE
When i use poe-api, poe doesn't respond.

### ISSUE THREAD

**Issue by Shirasawaz** [ading2210/poe-api#203]
When i use poe-api, poe doesn't respond.

https://github.com/ading2210/poe-api/assets/122033028/44fc456a-160a-44c0-83b4-074964fe14ae

**Comment by Colorzzzz** [comment]
logs
![21](https://github.com/ading2210/poe-api/assets/122033028/443dad05-c36a-4ad4-a1f1-aecd45009c2d)

**Comment by LanQian528** [comment]
> 

try claude2 with poe-api

**Comment by LanQian528** [comment]
this project has problems about the formkey.

**Comment by Kot-nikot** [comment]
The situation is similar when using model gpt-4.
If you use gpt-3.5 then everything works fine.

**Comment by isJN** [comment]
I feel that there have been some changes to the formkey of GPT4

**Comment by greengeckowizard** [comment]
yeah gpt 3.5 models seem to be working fine with the api wrapper but the gpt-4 models seem to be a bust atm I agree I think its the formkey of GPT4 models has been altered

**Comment by fkx816** [comment]
same problem

**Comment by Colorzzzz** [comment]
> The situation is similar when using model gpt-4. If you use gpt-3.5 then everything works fine.

u are right, everything works fine with gpt-3.5 or gpt-3.5-16k.

**Comment by Kot-nikot** [comment]
I specified the formkey manually and everything works ( formkey that I saved 2 days ago before the release of version 4.17 of poe-api, which means the algorithm for its formation has not changed) which means the algorithm for its formation has not changed.

**Comment by isJN** [comment]
Perhaps it is necessary to automatically obtain the corresponding account's formkey
